app/players/elves/elf.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC
+from app.players.player import Player
+
+
+class Elf(Player, ABC):
+    def __init__(self, nickname: str, musical_instrument: str) -> None:
+        super().__init__(nickname)
+        self.__musical_instrument = musical_instrument
```

Comment:
Do you know the difference between protected and private attributes? Fix in all similar cases

Reply:
roger that !